### DISCUSSION TITLE
How to use unlabeled data to train self supervised learning tasks

### DISCUSSION THREAD

**Discussion by majintao1994** [open-mmlab/mmselfsup#532] [How-to] [answered]
Hello, developer：
    I have been following the mmselfsup framework for some time. I am a beginner of self-monitoring learning. I would like to ask if there are any methods for training self-monitoring learning with unlabeled data in the document? Because I can't find it in the current document, thank you.

**Comment by tonysy** [comment, marked as answer]
Hi, we provide an example of using custom dataset in our latest version.
Please follow this discussion for more details.
https://github.com/open-mmlab/mmselfsup/discussions/510

**Comment by majintao1994** [comment]
okay, thank you

**Comment by fangyixiao18** [comment]
Convert to discussion, free to open a new one if there are any further problems.
